extensions/vscode/src/multiStepInputs/newDeployment.ts

```diff
@@ -266,7 +266,7 @@ export async function newDeployment(
   let credentialListItems: QuickPickItem[] = [];
 
   let entryPointListItems: QuickPickItemWithIndex[] = [];
-  let configDetails: ConfigurationDetails[] = [];
+  let configDetails: ConfigurationInspectionResult[] = [];
```

Comment:
I like what you did in `selectConfig.ts` where you renamed `configDetails` to `inspectionResults`. Can you update this to be refactored in the same way? I've been trying to keep the code as similar as possible, so that we can refactor it all into shared code, sometime in the future...